vformer/models/classification/vanilla.py

```diff
@@ -42,7 +60,7 @@ def __init__(
         self.pool = lambda x: x.mean(dim=1) if pool == "mean" else x[:, 0]
 
         if decoder_config is not None:
-            self.decoder = MLPDecoder(decoder_config, n_classes)
+            self.decoder = nn.Sequential(nn.Linear(latent_dim,decoder_config[0]),MLPDecoder(decoder_config, n_classes))
```

Comment:
I think just having an assert statement for the first dimension in the decoder config is better. We shouldn't add an extra layer to make it compatible.

Reply:
` assert latent_dim == decoder_config[0], "first item in ``decoder_config`` should be equal to ``latent_dim``" `
 This will work right?